chroot_setup

```diff
@@ -0,0 +1,440 @@
+#!/QOpenSys/pkgs/bin/python3
+
+import argparse
+import os
+import sys
+import shutil
+import logging
+import stat
+from string import Template
+from enum import Enum
+
+parser = argparse.ArgumentParser()
+# positional arguments
+parser.add_argument("chroot_directory", metavar="CHROOT_DIRECTORY", help="The location of the chroot to be created")
+parser.add_argument("chroot_type", metavar="CHROOT_TYPE", help="A list of .lst files to use for generating the chroot", nargs="?")
+# optional arguments
+parser.add_argument("-g", dest='globals', nargs='*', metavar="", help="Environment variables to set when creating the chroot")
+parser.add_argument("-i", dest='yum_installs', nargs='*', metavar="", help="A list of packages for yum to install in the chroot")
+parser.add_argument("-y", dest='auto_yes', action='store_true', help="Automatically answer yes to input prompts")
+parser.add_argument("-v", dest='verbose', action='store_true', help="Print verbose output when running")
+args = parser.parse_args()
+parser.parse_args()
+
+print("""
+  #####  #     # ######  ####### ####### #######
+ #     # #     # #     # #     # #     #    #   
+ #       #     # #     # #     # #     #    #   
+ #       ####### ######  #     # #     #    #   
+ #       #     # #   #   #     # #     #    #   
+ #     # #     # #    #  #     # #     #    #   
+  #####  #     # #     # ####### #######    #   
+                                                
+  #####  ####### ####### #     # ######         
+ #     # #          #    #     # #     #        
+ #       #          #    #     # #     #        
+  #####  #####      #    #     # ######         
+       # #          #    #     # #              
+ #     # #          #    #     # #              
+  #####  #######    #     #####  #
+""")
+
+CHROOT_DEBUG : bool = False
+
+print(args.chroot_directory)
+
+# :mkdir
+# Make a directory inside the chroot, as well as all intermediary directories
+def chroot_mkdir (dir: str):
+  print(f'os.makedirs({args.chroot_directory + dir}, exist_ok=True)')
+  if (CHROOT_DEBUG == False):
+    os.makedirs(args.chroot_directory + dir, exist_ok=True)
+
+# :ln
+# Create an absolute symlink. We don't have to go inside the chroot, since the
+# links are resolved as they are used. The target doesn't even have to exist,
+# Python won't check and therefore won't throw an error.
+def chroot_ln (line: str):
+  ln_command = line.split()
+  assert len(ln_command) == 2, ":ln action expects 2 arguments!"
+  target = ln_command[0]
+  link_name = ln_command[1]
```

Comment:
This can be done a bit more succinctly:

```suggestion
  target, link_name = ln_command
```

Reply:
Fixed up in 04ca7a33a71c868370b6315b537307b3984e03c2